build.gradle

```diff
@@ -29,12 +29,11 @@ sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 //def depJFreeChart = 'jfreechart:jfreechart:1.0.19-antialiasing'
-def depJFreeChart = 'org.jfree:jfreechart:1.0.19'
```

Comment:
It would probably be cleaner to leave depJFreeChart variable around and set it to the files(...) used below.